package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cordova-plugin-inappbrowser",
-  "version": "3.2.1-dev",
+  "version": "3.2.2-dev",
```

Comment:
Please revert the version changes again.

Reply:
I reverted all version changes.